rca/project_styleguide/templates/patterns/molecules/jump-nav/jump-nav.html

```diff
@@ -0,0 +1,19 @@
+<div class="jump-nav tab-nav {{ modifier }} {% if sticky %}sticky{% endif %}">
+    <div class="jump-nav__layout tab-nav__layout grid">
+        <div class="jump-nav__heading">
+            <div class="jump-nav__heading-label">
+                <span class="jump-nav__pre-stuck-heading">Jump to</span>
+                <span class="jump-nav__stuck-heading">{{ title }}</span>
```

Comment:
guessing there will always be a title so this doesn't need a conditional around it?

Reply:
I'll add one in, won't do any harm :) Cheers